lldb/source/Target/TargetProperties.td

```diff
@@ -360,3 +360,9 @@ let Definition = "thread" in {
     DefaultUnsignedValue<600000>,
     Desc<"Maximum number of frames to backtrace.">;
 }
+
+let Definition = "language" in {
+  def EnableFilterForLineBreakpoints: Property<"enable-filter-for-line-breakpoints", "Boolean">,
```

Comment:
Is there a naming convention around boolean properties? Should this be prefixed with enable-, or not? It seems possibly redundant to me. Maybe the name should be `filter-for-line-breakpoints` and the values `enable` and `disable`? I'm not strongly opposed to the name, but it did strike me as a little unconventional.

Reply:
Current practice has all "enable"-like properties using boolean values, and most of them mentioning "enable" in the actual setting flag, though there are exception to this.